diff --git a/NBXplorer.Client/NBXplorer.Client.csproj b/NBXplorer.Client/NBXplorer.Client.csproj
index c55c053..e19d644 100644
--- a/NBXplorer.Client/NBXplorer.Client.csproj
+++ b/NBXplorer.Client/NBXplorer.Client.csproj
@@ -14,9 +14,6 @@
 		<RepositoryType>git</RepositoryType>
 		<LangVersion>8.0</LangVersion>
 	</PropertyGroup>
-      	<ItemGroup Condition=" '$(Configuration)' == 'Release' ">
-		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-	</ItemGroup>
         <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
              	<PublishRepositoryUrl>true</PublishRepositoryUrl>
 		<EmbedUntrackedSources>true</EmbedUntrackedSources>
@@ -28,5 +25,6 @@
 		<PackageReference Include="NBitcoin.Altcoins" Version="2.0.21" />
 		<PackageReference Include="NicolasDorier.StandardConfiguration" Version="1.0.0.18" />
 		<PackageReference Include="System.Net.WebSockets.Client" Version="4.3.2" />
+ 	        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
 	</ItemGroup>
 </Project>
